cmd/brimcap/analyze/command.go

```diff
@@ -46,16 +48,24 @@ type Command struct {
 	*root.Command
 	analyzecli.Display
 	analyzeflags analyzecli.Flags
+	nostats      bool
 	out          outputflags.Flags
 }
 
 func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
 	c := &Command{Command: parent.(*root.Command)}
 	c.analyzeflags.SetFlags(f)
 	c.out.SetFlags(f)
+	f.BoolVar(&c.nostats, "nostats", false, "do not display stats in stderr")
 	return c, nil
 }
 
+// json:
+// - always display stats in stderr (except if -nostats is enabled)
+// status line display stats iff:
+// - -o is a file: display stats
+// - -o is stdout and stdout is NOT a terminal: display stats
+
```

Comment:
Put this adjacent to the code it describes so it'll get updated if that code changes.

Reply:
Sure!